zipline/algorithm.py

```diff
@@ -304,6 +306,12 @@ def __init__(self, *args, **kwargs):
                 trading_calendar=self.trading_calendar,
             )
 
+        # Ensure that capital base hasn't been overridden to an invalid value
+        # This test duplicates that from qexec/algoproxy, but targets users
```

Comment:
Is this comment still true?

Reply:
No, good catch. Forgot that I removed the algoproxy code